src/main/java/seedu/address/model/student/academic/Academic.java

```diff
@@ -35,7 +35,7 @@ public String getFormattedAttendance() {
         String result = "";
         int index = 1;
         for (Attendance singleAttendance : attendance) {
-            result = result + index + ". " + singleAttendance.getFormattedAttendance() + "\n";
+            result = result + index + ". " + singleAttendance + "\n";
             index++;
         }
```

Comment:
Sorry to bother you with this, but do you think you could just do the same for Questions to have them numbered in the GUI?

Reply:
Sure